Squeeze v0.20.1.r3

VauerSqueeze opened the pull request:
dev/PIPETD-598/TDCOMP-Nuke_Versionning-Added_a_template_switch_to_toggle_between_different_project_rules_for_versioning_Nuke_files

**Description**
- Added a template switch to toggle between different project rules for versioning Nuke files

**Release Notes**
- Added a template  switch to toggle between different project rules for versioning Nuke files

**Tests**

**Jira**
[PIPETD-590](https://squeezestudio.atlassian.net/browse/PIPETD-598)

VauerSqueeze committed:
dev/PIPETD-598/TDCOMP-Nuke_Versionning-Added_a_template_switch_to_toggle_between_different_project_rules_for_versioning_Nuke_files
**Description**
- Added a template switch to toggle between different project rules for versioning Nuke files

**Release Notes**
- Added a template  switch to toggle between different project rules for versioning Nuke files

**Tests**

**Jira**
[PIPETD-590](https://squeezestudio.atlassian.net/browse/PIPETD-598)

VauerSqueeze committed:
PIPETD-598/Added_a_template_switch_to_toggle_between_different_project_rules_for_versioning_Nuke_files
**Description**
- Added a template switch to toggle between different project rules for versioning Nuke files

**Release Notes**
- Added a template  switch to toggle between different project rules for versioning Nuke files

**Tests**

**Jira**
[PIPETD-590](https://squeezestudio.atlassian.net/browse/PIPETD-598)

VauerSqueeze committed:
Update bundle.py

cblanchette97 commented:
Important note for this pull request. 
Since our sgtk config now points to the `squeeze_v0.20.1.r2` branch, we must not merge the branch `squeeze_v0.20.1.r3` into it. If we approve the changes, we simply need to make our sgtk config point to the `squeeze_v0.20.1.r3` branch and keep `squeeze_v0.20.1.r2` intact in case we need to revert to it!

VauerSqueeze committed:
dev/PIPETD-598/TDCOMP-Nuke_Versionning-Added_a_template_switch_to_toggle_between_different_project_rules_for_versioning_Nuke_files
**Description**
- Added a template switch to toggle between different project rules for versioning Nuke files

**Release Notes**
- Added a template  switch to toggle between different project rules for versioning Nuke files

**Tests**

**Jira**
[PIPETD-590](https://squeezestudio.atlassian.net/browse/PIPETD-598)

VauerSqueeze commented:
This branch is used since the merge release of [branch](http://gitlab.squeeze.lan/prod/sgtk/-/merge_requests/416) on sgtk
